src/core/server/graph/context.ts

```diff
@@ -86,6 +89,8 @@ export default class GraphContext {
   public readonly signingConfig?: JWTSigningConfig;
   public readonly user?: User;
 
+  public readonly dataContext: DataContext;
```

Comment:
Following the feedback around renaming the `DataContext` class to `Connectors`, similarrly this would then become:

```suggestion
  public readonly connectors: Connectors;
```